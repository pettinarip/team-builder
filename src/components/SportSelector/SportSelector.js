import React from 'react'

const SportSelector = () => {
  return (
    <div className='settings-field-wrapper'>
      <label className='settings-field-heading'>Sport</label>
      <div className='settings-field'>
        <div className='list'>
          <div className='item selected'>
            <div
              className='layout-description'
            >
              Football
            </div>
          </div>
        </div>
      </div>
    </div>
  )
}

SportSelector.propTypes = {}

export default SportSelector
