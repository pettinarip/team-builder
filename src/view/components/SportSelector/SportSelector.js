import React from 'react'
import SettingField from '../SettingField'

const SportSelector = () => {
  return (
    <SettingField title='Sport'>
      <div className='list'>
        <div className='item selected'>
          <div
            className='layout-description'
          >
            Football
          </div>
        </div>
      </div>
    </SettingField>
  )
}

SportSelector.propTypes = {}

export default SportSelector
