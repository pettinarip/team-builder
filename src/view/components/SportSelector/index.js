import SportSelector from './SportSelector'

export default SportSelector
