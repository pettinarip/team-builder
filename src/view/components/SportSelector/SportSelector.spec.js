import React from 'react'
import { shallow } from 'enzyme'
import SportSelector from './SportSelector'

describe('SportSelector component', () => {
  it('should render the component properly', () => {
    const component = shallow(<SportSelector />)
    expect(component).toMatchSnapshot()
  })
})
