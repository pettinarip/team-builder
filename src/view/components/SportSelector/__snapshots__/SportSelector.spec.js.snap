// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SportSelector component should render the component properly 1`] = `
<SettingField
  title="Sport"
>
  <div
    className="list"
  >
    <div
      className="item selected"
    >
      <div
        className="layout-description"
      >
        Football
      </div>
    </div>
  </div>
</SettingField>
`;
